src/components/MapExplorer.js

```diff
@@ -30,6 +30,7 @@ import React, {
 import {useTranslation} from 'react-i18next';
 import {useHistory} from 'react-router-dom';
 import {animated, useSpring} from 'react-spring';
+import {Swipeable} from 'react-swipeable';
```

Comment:
Since we use React hooks everywhere else in the code, do you mind replacing this with `useSwipeable`?

Reply:
Sure, I will make the changes.